backend/SlothOrganizer/SlothOrganizer.Services/Auth/UserVerification/UserVerificationService.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using Microsoft.AspNetCore.WebUtilities;
+using Microsoft.Extensions.Configuration;
+using SlothOrganizer.Domain.Exceptions;
+using SlothOrganizer.Services.Abstractions.Auth.UserVerification;
+using SlothOrganizer.Services.Abstractions.Email;
+using SlothOrganizer.Services.Abstractions.Utility;
+
+namespace SlothOrganizer.Services.Auth.UserVerification
+{
+    public class UserVerificationService : IUserVerificationService
```

Comment:
Renamed this service to EmailNotificationService, and it's interface to INotificationServcie, since it's responsibility is basically sending notifications to user via email